### DISCUSSION TITLE
SmallRye and Kafka - Quotes application fails with HTTP 404

### DISCUSSION THREAD

**Discussion by chainhead** [quarkusio/quarkus#28882] [Q&A] [answered]
I have followed this [article](https://quarkus.io/guides/kafka-reactive-getting-started) step-by-step to execute it with Docker Compose. I am able to launch the HTML `http://localhost:8080/quotes.html`, however upon clicking `Request Quote` there is no response. The debug tools in the browser shows that it has encountered a `404`.

What is missing?

```bash
POST http://localhost:8080/quotes/request
[HTTP/1.1 404 Not Found 11ms]

	
POST
	http://localhost:8080/quotes/request
Status
404
Not Found
VersionHTTP/1.1
Transferred45 B (0 B size)
Referrer Policystrict-origin-when-cross-origin
Request PriorityHighest

```

[a comment by an automated account is not shown]

**Comment by chainhead** [comment, marked as answer]
Please ignore; it is working now. It is a bit difficult to swap between two projects and there were no instructions on which `application.properties` had to be edited.

**Reply by gsmet** [reply]
Could you clarify what's missing in the guides?

**Comment by ozangunalp** [comment]
I'll definitely take a look to clarify things.
